spec/jobs/data_import_job_spec.rb

```diff
@@ -13,9 +14,23 @@
 
   it 'imports data into the account' do
     csv_length = CSV.parse(data_import.import_file.download, headers: true).length
+
     described_class.perform_now(data_import)
     expect(data_import.account.contacts.count).to eq(csv_length)
     expect(data_import.reload.total_records).to eq(csv_length)
     expect(data_import.reload.processed_records).to eq(csv_length)
   end
+
```

Comment:
Let's add specific test cases that we should have included earlier. 

- If existing email contact exists the data gets updated
- if existing phone number contact exists data gets updates
-  imported contacts count / specific attributes are updated correctly

## edge cases
- csv contains a row with email 'xyc` and phone number `123` . the database has contact 1 with email `xyc` and contact 2 with phone_number `123`  .. which contact gets updated ?